static/src/javascripts/projects/common/modules/async-call-merger.spec.ts

```diff
@@ -0,0 +1,35 @@
+import Chance from 'chance';
+import { noop } from 'lib/noop';
+import type { TargetCallback } from './async-call-merger';
+import { mergeCalls } from './async-call-merger';
+
+const chance = new Chance();
+const RESULT = chance.string();
```

Comment:
Is it important that this is a random string each time? I wonder if it would make the test simpler to just hardcode this to a given string giving a predictable value for every test run. (Again, I know this was carried over from the js test so totally fine to leave as is).

Reply:
This also means we can drop the `@types/chance` deps, so I’ll do that instead. I was focussing more on the generics here, but it’s worth improving the test cases, too!